spec/models/equipment_model_spec.rb

```diff
@@ -286,41 +339,62 @@
 
     context 'methods involving reservations' do
       # @model and @category are already set.
+      ACTIVE = [:valid_reservation, :checked_out_reservation]
```

Comment:
We do, it just gets handled separately because it affects availability differently than these statuses